test/core/utils/preload-config.js

```diff
@@ -0,0 +1,80 @@
+describe('axe.utils.preloadConfig', function () {
+	'use strict';
+
+	it('should be a function', function () {
+		assert.isFunction(axe.utils.preloadConfig);
+	});
+
+	it('should return default preload configuration if no preload options', function () {
+		var actual = axe.utils.preloadConfig({});
+		var expected = {
```

Comment:
This seems fragile. Better to just pull in the default and test that 1) it has the properties you need from it 2) it matches what's returned.